Packs/AnomaliThreatStreamFeed/Integrations/AnomaliThreatStreamFeed/README.md

```diff
@@ -0,0 +1,59 @@
+Use the Anomali ThreatStream Feed Integration to fetch indicators from Anomali ThreatStream.
+
+## Configure Anomali ThreatStream Feed in Cortex
+
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| Fetch indicators | Enable this checkbox to automatically pull indicators from Anomali ThreatStream at regular intervals. |  |
+| Fetch by | Fetch by the modification or creation time of the indicators. | True |
+| Server URL (e.g., https://www.test.com) | Confirm that the pre-filled URL matches the correct API endpoint for your Anomali ThreatStream instance. | True |
+| Username |  | True |
+| API Key |  | True |
+| Feed Fetch Interval |  | False |
+| Confidence Threshold | Will only return indicators above the confidence threshold. | False |
+| Source Reliability | Reliability of the source providing the intelligence data | True |
+| Traffic Light Protocol Color | The Traffic Light Protocol \(TLP\) designation to apply to indicators fetched from the feed. Indicator's TLP will override default value. | False |
```

Comment:
Traffic Light Protocol Color | The Traffic Light Protocol \(TLP\) designation to apply to indicators fetched from the feed. Indicator's TLP will override the default value. | False |